core/src/main/java/de/gurkenlabs/litiengine/entities/Entity.java

```diff
@@ -68,6 +68,8 @@ public abstract class Entity implements IEntity, EntityRenderListener, Tweenable
   @TmxProperty(name = MapObjectProperty.RENDERTYPE)
   private RenderType renderType;
 
+  private float opacity = 1f;
```

Comment:
While the idea of setting the drawing opacity is nice, I don't approve how it's implemented:
We should keep rendering-specific stuff separate from the Entity logic. 
The preferred way to achieve this visual effect would be to apply an ImageEffect in the Entity's AnimationController.

Please remove all Entity opacity related code from this PR.

Reply:
```suggestion
```